cli/sync.go

```diff
@@ -149,6 +153,10 @@ var SyncUnmarkBadCmd = &cli.Command{
 		defer closer()
 		ctx := ReqContext(cctx)
 
+		if cctx.NArg() != 1 {
```

Comment:
this duplicates a block below (`if !cctx.Args().Present()`), but you could bring it up to the top of the function with the message and use `IncorrectNumArgsWithHint` to preserve it